policies/releasestrat.html

```diff
@@ -20,27 +20,54 @@ <h5>
 	<div class="entry-content">
 
 	  <p>
-	  As of release 1.0.0 the OpenSSL versioning scheme was improved
-	  to better meet developers' and vendors' expectations. Letter
-	  releases, such as 1.0.2a, exclusively contain bug and security
-	  fixes and no new features. Minor releases that change the
-	  last digit, e.g. 1.1.0 vs. 1.1.1, can and are likely to
-	  contain new features, but in a way that does not break binary
-	  compatibility. This means that an application compiled and
-	  dynamically linked with 1.1.0 does not need to be recompiled
-	  when the shared library is updated to 1.1.1. It should be
-	  noted that some features are transparent to the application
-	  such as the maximum negotiated TLS version and cipher suites,
-	  performance improvements and so on. There is no need to
-	  recompile applications to benefit from these features.</p>
+	    As of release 3.0.0, the OpenSSL versioning scheme is changing
+	    to a more contemporary format: MAJOR.MINOR.PATCH
+	  </p>
+
+	  <p>
+	    With this format, API/ABI compatibility will only be guaranteed
+	    for the same MAJOR version number. Previously we guaranteed
+	    API/ABI compatibility across the same MAJOR.MINOR combination.
+	  </p>
+
+	  <p>
+	    This more closely aligns with the expectations of users who are
+	    familiar with semantic versioning.  However, we have not adopted
+	    semantic versioning in the strict sense of its rules, because it
+	    would mean changing our current LTS policies and practices.
```

Comment:
See for example https://github.com/openssl/openssl/issues/5042#issuecomment-444204003 and ff.

Reply:
I saw a statement of "we aren't going to do it the semver way", but no rationale for why. Saying because its current policy is a head-scratcher, because the policy is changing: API breaks are no longer allowed in MINOR. Not allowing API additions in PATCH could occur at the same time, and that would be all the way to semver. LTS policies are not covered by semver, of course, those are project specific.

Node has LTS releases and uses semver. Within a LTS-designated major, if we decide to add accessors (or something of the like), we bump the minor. No big deal. Not all new features go back into LTS. They need to be considered stable, and exist in a current release for some time (for bug shakedown). Eventually LTS goes into "maintenance mode", where we don't do anything except critical bug fixes. I don't think node's stability requirements are fundamentally that different from OpenSSL, other than that your LTS are much longer (a good thing, thank you). What we do is described here, in case its of interest: https://github.com/nodejs/Release#release-plan

Anyhow, I'm done kibitzing. The policy described here will work for us as OpenSSL consumers.